src/components/atoms/Input/Input.styled.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import typography from 'styles/typography';
+
+export const InputWrapper = styled.div`
+  box-sizing: border-box;
+  height: 52px;
+  border-radius: 4px;
+  border: 1px solid ${colors.background.Neutral80};
+  background: ${colors.background.NeutralWhite};
+  display: flex;
+  align-items: center;
+  position: relative;
+`;
+
+export const StyledInput = styled.input<{ icon: any }>`
```

Comment:
Please do something with this type. Any is not acceptable because of many bugs related